sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_backup_client.py

```diff
@@ -104,7 +116,20 @@ def begin_restore(self, folder_url, sas_token, **kwargs):
 
         status_response = None
         if continuation_token:
-            status_response = _get_status_from_continuation_token(continuation_token, self._client._client)
+            try:
+                job_id = _parse_status_url(base64.b64decode(continuation_token.encode()).decode("ascii"))
+            except:  # pylint: disable=broad-except
+                raise ValueError("The provided continuation_token is malformed. A base64-encoded URL is expected")
```

Comment:
This implies a user could, or in some cases should, construct a token.
```suggestion
                raise ValueError("The provided continuation_token is malformed.")
```